website/management/commands/base_lab_seeder.py

```
@@ -0,0 +1,55 @@
+from django.core.management.base import BaseCommand, CommandError
+
+from website.models import Labs, TaskContent, Tasks
+
+
+class LabSeederCommand(BaseCommand):
+    lab_name = None  # Override in subclass
+    tasks_data = []  # Override in subclass
```

Ruff (0.14.3)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)